File: health-mobile/health.mobile/Models/Metric.cs
using System;
using Newtonsoft.Json;

namespace health.mobile.Models
{
    public class Metric
    {
        [JsonProperty(PropertyName = "agent_name")]
        public string AgentName { get; set; }
        [JsonProperty(PropertyName = "metric_name")]
        public string Name { get; set; }
        [JsonProperty(PropertyName = "value")]
        public string Value { get; set; }
        [JsonProperty(PropertyName = "state")]
        public string State { get; set; }
        [JsonProperty(PropertyName = "date")]
        public DateTime Date { get; set; }
        [JsonIgnore]
        public string DateString {
            get {
                return Date.ToString("dd.MM.yyyy HH:mm:ss");
            }
        }

        public Metric()
        {
        }
    }
}

File: health-client/src/main/java/com/epam/healthmonitor/android/model/Metric.kt
package com.epam.healthmonitor.android.model;

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.Date

class Metric(@JsonProperty(value = "agent_name")
             val agentName: String,
             @JsonProperty(value = "date")
             val date: Date,
             @JsonProperty(value = "metric_name")
             val metricName: String,
             @JsonProperty(value = "state")
             val state: String,
             @JsonProperty(value = "value")
             val value: String
)